src/examples/forms/file-upload/FileList.tsx

```diff
@@ -10,54 +10,75 @@ import { FileList, File } from '@zendeskgarden/react-forms';
 import { Progress } from '@zendeskgarden/react-loaders';
 import { Row, Col } from '@zendeskgarden/react-grid';
 import { KEY_CODES } from '@zendeskgarden/container-utilities';
-import { Tooltip } from '@zendeskgarden/react-tooltips';
 
 const handleClick = () => alert('File dismissed via mouse');
 
-const handleKeyDown = (e: React.KeyboardEvent<any>) => {
+const handleFileKeyDown = (e: React.KeyboardEvent<any>) => {
   if (e.keyCode === KEY_CODES.DELETE || e.keyCode === KEY_CODES.BACKSPACE) {
     e.preventDefault();
     alert('File dismissed via keyboard');
   }
 };
 
+const handleCloseKeyDown = (e: React.KeyboardEvent<any>) => {
+  const KEYS = [KEY_CODES.SPACE, KEY_CODES.ENTER, KEY_CODES.DELETE, KEY_CODES.BACKSPACE];
+
+  if (KEYS.includes(e.keyCode)) {
+    e.preventDefault();
+    alert('File dismissed via keyboard');
+  }
+};
+
 const Example = () => (
   <Row justifyContent="center">
     <Col sm={5}>
       <FileList>
         <FileList.Item>
-          <File type="generic" aria-label="Generic file" tabIndex={0} onKeyDown={handleKeyDown}>
+          <File type="generic" aria-label="Generic file" tabIndex={0} onKeyDown={handleFileKeyDown}>
             Garden file
-            <Tooltip content="Stop upload">
-              <File.Close aria-label="close" onClick={handleClick} tabIndex={-1} />
-            </Tooltip>
+            <File.Close
+              aria-label="Press delete to remove"
```

Comment:
Please retain the original tooltip intent throughout. In this case the label should be "Stop upload" as that is the intended design for what we expect users to do with the `File.Close` interaction.